app/views/course/experience_points_records/_experience_points_specific.json.jbuilder

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
```

Comment:
Remove `_response_experience_points_reason.html.slim` and `_submission_experience_points_reason.html.slim` as well?